addons/html_editor/static/src/main/powerbox/powerbox_plugin.js

```diff
@@ -26,7 +26,7 @@ import { Powerbox } from "./powerbox";
 function target(selection) {
     const node = selection.anchorNode;
     const el = node.nodeType === Node.ELEMENT_NODE ? node : node.parentElement;
-    if (selection.inEditable && (el.tagName === "DIV" || el.tagName === "P") && isEmpty(el)) {
+    if (selection.inEditable && (el.tagName === "DIV" || el.tagName === "P") && isEmptyBlock(el)) {
```

Comment:
This change does not seem to reflect a change that was done in the original PR for `web_editor`. Why was this change required?

Reply:
@dmo-odoo I have made this change because When the enter key is pressed on an element like `p` or `quote`, the newly created tag does not display the placeholder. This happens because the `<br>` tag in the newly created tag has the attribute `data-last-history-steps`. This attribute causes the `innerHTML` comparison in the `isEmpty` method to return false, which prevents the placeholder from being applied to the tag.

Reply:
@FrancoisGe This one in particular sounds like a bug, right?

Reply:
Yes, you're right, it's a mistake

Reply:
Thanks @FrancoisGe, then I think the other possibly wrong usage of `isEmpty` instead of `isEmptyBlock` is likely to be an oversight as well. cc @dhba-odoo I'm resolving those two comments then, you were right to change it to `isEmptyBlock`, good job!